lib/views/backend/spree/admin/shared/_navigation_footer.html.erb

```diff
@@ -1,9 +1,16 @@
 <% if spree_current_user %>
   <ul id="login-nav" class="admin-login-nav">
     <li data-hook="user-account-link">
-      <%= link_to spree.edit_admin_user_path(spree_current_user) do %>
-        <i class='fa fa-user'></i>
-        <%= spree_current_user.email %>
+      <% if can?(:admin, try_spree_current_user) %>
+        <%= link_to spree.edit_admin_user_path(try_spree_current_user) do %>
+          <i class='fa fa-user'></i>
+          <%= try_spree_current_user.email %>
+        <% end %>
+      <% else %>
+        <%= link_to spree.admin_path do %>
```

Comment:
Linking to the root admin_path might be more confusing than having no link at all